tests/scripts/all.sh

```diff
@@ -1120,6 +1120,70 @@ component_test_cmake_out_of_source () {
     unset MBEDTLS_ROOT_DIR
 }
 
+component_test_basic_mbed_os_conf () {
+msg "build+test: Mbed OS default configuration"
+    scripts/config.pl unset MBEDTLS_NET_C
+    scripts/config.pl unset MBEDTLS_TIMING_C
+    scripts/config.pl unset MBEDTLS_HAVE_TIME_DATE
+    scripts/config.pl unset MBEDTLS_FS_IO
+    scripts/config.pl unset MBEDTLS_PSA_ITS_FILE_C # requires a filesystem
+    scripts/config.pl unset MBEDTLS_PSA_CRYPTO_STORAGE_C # requires PSA ITS
+    #scripts/config.pl set MBEDTLS_NO_PLATFORM_ENTROPY Need some entropy
```

Comment:
Why is this line commented out?

Reply:
Because we need some entropy sources, otherwise `psa_crypto_init()` will fail